include/SFML/Graphics/Shape.hpp

```diff
@@ -181,6 +199,16 @@ class SFML_GRAPHICS_API Shape : public Drawable, public Transformable
     ////////////////////////////////////////////////////////////
     float getOutlineThickness() const;
 
+    ////////////////////////////////////////////////////////////
+    /// \brief Get the limit on the ratio between miter length and outline thickness
+    ///
+    /// \return Limit on the ratio between miter length and outline thickness
+    ///
+    /// \see setMiterLimit
+    ///
+    ////////////////////////////////////////////////////////////
+    float getMiterLimit() const;
```

Comment:
I want to challenge the "Miter" name usage. I believe it makes on a "domain" level a lot of sense to call it that, but as a simple users that wants to draw shapes, it's not easy to understand what "miter" or "miter limit" means.

Just searching the term "miter" leads to the "tall headdress of a bishop" or "joint made between two pieces of wood". You really have to search for "miter limit" or similar until you start getting documentation from SVG/canvas.

I don't have a better name ready just yet, and am also open to using "miter limit", but maybe we can brainstorm a bit, if there's something that describes it in a way that doesn't necessarily require a documentation study or web search.

Reply:
The PR name uses the word `bevel` which I think is a good descriptor. This parameter controls how far the outline corner can stick out before it starts being beveled.

Reply:
I believe "miter limit" it is the correct name because it is already used by SVG as well as [cairo library](https://www.cairographics.org/manual/cairo-cairo-t.html#cairo-set-miter-limit). It would be more confusing to come up with some other name. I agree the meaning is not obvious and it deserves more documentation. I will work on it.

Reply:
I understand that it's used like that within the domain of SVG/graphics, but we shouldn't just stop there and do as everyone else does. The term is not intuitively understood and as such it deserves a bit of a brainstorming in my opinion.

Reply:
I asked a bit on social media, to see what people would more intuitively call this.

- Two mentioned miter, but more because they've learned the word from graphics applications.
- sanitizeOutline
- setOutlineExtensionLimit
- limitOutlineExtension
- clamp
- cutting corners
- corner style
- capping (used in Godot)
- Corner length
- Cap
- Clip
- Limit
- bevel

https://swiss.social/@darkcisum/111391908664038204 
https://twitter.com/DarkCisum/status/1723282675878789343

One interesting thought that came out of these discussions is that this could in theory be generalized further and allow for different "styling" of the corners. I wouldn't to do this right now of course, but it might influence the thinking of naming.